src/ui/src/builder/builderManager.ts

```diff
@@ -406,6 +406,22 @@ export function generateBuilderManager() {
 		return state.value.logEntries;
 	};
 
+	const activeBlueprintRunId = computed(() => {
+		const logEntries = getLogEntries();
+		const runId =
+			logEntries.find((entry) => {
+				return !!entry.blueprintExecution;
+			})?.blueprintExecution?.runId ?? null;
```

Comment:
If we don't have the `runId`, we could early return, no ?
```suggestion
		const runId =
			logEntries.find((entry) => {
				return !!entry.blueprintExecution;
			})?.blueprintExecution?.runId;
	    if (runId === undefined) return null;
```